src/main/resources/data-mapping.json

```diff
@@ -718,6 +718,466 @@
+          "dataSource": {
+            "default": "Awaiting Receipt"
+          }
+        },
+        {
+          "field": "PO_LINE_PAYMENT_STATUS",
+          "dataSource": {
+            "default": "Awaiting Payment"
+          }
+        },
+        {
+          "field": "SOURCE_ID",
+          "dataSource": {
+            "default": "Loaded from Vendor (API)"
+          }
+        },
+        {
+          "field": "VENDOR_REF_NO_TYPE",
+          "dataSource": {
+            "default": "Supplier's unique order line reference number"
+          }
+        }
+      ]
+    },
+    {
+      "orderType": "ListedElectronicSerial",
+      "mappings": [
+        {
+          "field": "TITLE",
+          "dataSource": {
+            "from": "//datafield[@tag='245']/*",
+            "combinator": "concat"
+          }
+        },
+        {
+          "field": "PUBLISHER",
+          "dataSource": {
+            "from": "//datafield[@tag='260']/subfield[@code='b']"
+          }
+        },
+        {
+          "field": "FUND_CODE",
+          "dataSource": {
+            "from": "//FundCode",
+            "translation": "lookupFundId"
+          }
+        },
+        {
+          "field": "INSTRUCTIONS",
+          "dataSource": {
+            "from": "//OrderNotes"
+          }
+        },
+        {
+          "field": "QUANTITY",
+          "dataSource": {
+            "from": "//Quantity",
+            "default": "1",
+            "translation": "toInteger"
+          }
+        },
+        {
+          "field": "VENDOR_REF_NO",
+          "dataSource": {
+            "from": "//YBPOrderKey"
+          }
+        },
+        {
+          "field": "CREATED_DATE",
+          "dataSource": {
+            "from": "//OrderPlaced",
+            "translation": "toDate"
+          }
+        },
+        {
+          "field": "LIST_PRICE",
+          "dataSource": {
+            "from": "//ListPrice/Amount",
+            "default": "0",
+            "translation": "toDouble"
+          }
+        },
+        {
+          "field": "ESTIMATED_PRICE",
+          "dataSource": {
+            "from": "//ListPrice/Amount",
+            "defaultMapping": {
+              "dataSource": {
+                "from": "//NetPrice/Amount|//Quantity",
+                "combinator": "multiply",
+                "fromOtherField": "LIST_PRICE",
+                "translation": "toDouble"
+              }
+            },
+            "translation": "toDouble"
+          }
+        },
+        {
+          "field": "CURRENCY",
+          "dataSource": {
+            "from": "//ListPrice/Currency",
+            "default": "USD"
+          }
+        },
+        {
+          "field": "ACCESS_PROVIDER",
+          "dataSource": {
+            "from": "//PurchaseOrder/VendorPOCode",
+            "translation": "lookupVendorId"
+          }
+        },
+        {
+          "field": "NOTE_FROM_VENDOR",
+          "dataSource": {
+            "from": "//PurchaseOrder/VendorCode"
```

Comment:
Should it be //PurchaseOption rather than //PurchaseOrder?

Reply:
It should. I assume these were copied from the `ListedElectronicMonograph` mapping, which blindly copied from `PostGobiOrdersHelper`. It is `//PurchaseOrder/...` there. @KVupp, can you fix the `ListedElectronicMonograph` mappings for these values as well?